docs/hub-tutorials/join-mainnet.md

```diff
@@ -215,16 +189,169 @@ enable = true
+# header hash obtained from a trusted source, and a period during which validators can be trusted.
+#
+# For Cosmos SDK-based chains, trust_period should usually be about 2/3 of the unbonding time (~2
+# weeks) during which they can be financially punished (slashed) for misbehavior.
+rpc_servers = "https://rpc-cosmoshub.keplr.app:443,https://rpc.cosmos.network:443"
+trust_height = 8959784
+trust_hash = "3D8F12EA302AEDA66E80939F7FC785206692F8B6EE6F727F1655F1AFB6A873A5"
+trust_period = "168h0m0s"
+```
+
+Start Gaia to begin state sync. It may take take some time for the node to acquire a snapshot, but the command and output should look similar to the following:
+
+```bash
+$ gaiad start --x-crisis-skip-assert-invariants
+
+...
+
+> INF Discovered new snapshot format=1 hash="0x000..." height=8967000 module=statesync
+
+...
+
+> INF Fetching snapshot chunk chunk=4 format=1 height=8967000 module=statesync total=45
+> INF Applied snapshot chunk to ABCI app chunk=0 format=1 height=8967000 module=statesync total=45
+```
+
+Once state sync successfully completes, the node will begin to process blocks normally. If state sync fails and the node operator encounters the following error:  `State sync failed err="state sync aborted"`, either try restarting `gaiad` or running `gaiad unsafe-reset-all` (make sure to backup any configuration and history before doing this).
+
+### Quicksync
+Quicksync.io offers several  daily snapshots of the Cosmos Hub with varying levels of pruning (`archive` 1.4TB, `default` 540GB, and `pruned` 265GB). For downloads and installation instructions, visit the [Cosmos Quicksync guide](https://quicksync.io/networks/cosmos.html).
+
+
+## Snapshots
+Saving and serving snapshots helps nodes rapidly join the network.
+
+To contribute snapshots to the network, `snapshot-interval` needs to be configured in `~/.gaia/config/app.toml`. The Cosmos Hub recommends setting this value to `1000`. This value must match `pruning-keep-every` in `config.toml`.
+
+In `app.toml`
+```
+###############################################################################
+###                        State Sync Configuration                         ###
+###############################################################################
+
+# State sync snapshots allow other nodes to rapidly join the network without replaying historical
+# blocks, instead downloading and applying a snapshot of the application state at a given height.
+[state-sync]
+
+# snapshot-interval specifies the block interval at which local state sync snapshots are
+# taken (0 to disable). Must be a multiple of pruning-keep-every.
+snapshot-interval = 1000
+
+# snapshot-keep-recent specifies the number of recent snapshots to keep and serve (0 to keep all).
+snapshot-keep-recent = 10
+```
+
+It is highly recommended that node operators use the same value for snapshot-interval in order to aid snapshot discovery. Discovery is easier when more nodes are serving the same snapshots.
+
+
+## Hardware
```

Comment:
this feels like it should go near the top of the document, as it's a logical prerequisite.

Reply:
Agreed. We've since updated gaia to enable snapshotting with these values as the default so the wording will have to change and be moved to the top of this block